### PR TITLE
Add a cvar for displaying displacer's earth target entities

### PR DESCRIPTION
**Pull request by hobokenn**
image

### CODE REVIEW

File: BunnymodXT/modules/ServerDLL.hpp [hunk 1]
```diff
@@ -55,8 +55,9 @@ class ServerDLL : public IHookableDirFilter
 
 	std::vector<const edict_t *> GetUseableEntities(const Vector &origin, float radius) const;
 	std::vector<const Vector *> GetNodePositions() const;
+	std::vector<const Vector *> GetDisplacerTargets() const;
 	bool GetNihilanthInfo(float &health, int &level, int &irritation, bool &recharger, int &nspheres, int &sequence, float &frame) const;
-
+
```
Comment (YaLTeR) [inline BunnymodXT/modules/ServerDLL.hpp:60]: nice

**Comment by YaLTeR** [comment]
Thanks

File: BunnymodXT/modules/ServerDLL.hpp (hunk 1 above)
Comment (hobokenn) [reply BunnymodXT/modules/ServerDLL.hpp:60]: didn't show as a diff in visual studio 🤷

Comment (YaLTeR) [reply BunnymodXT/modules/ServerDLL.hpp:60]: Maybe it ignores whitespace by default? I know in vscode there's definitely a button for that

**Pull request merged**